mindsdb/integrations/handlers/lindorm_handler/__init__.py

```diff
@@ -0,0 +1,26 @@
+from mindsdb.integrations.libs.const import HANDLER_TYPE
+
+from .__about__ import __description__ as description
+from .__about__ import __version__ as version
+
+try:
+    from .lindorm_handler import LindormDBHandler as Handler
+
+    import_error = None
+except Exception as e:
+    Handler = None
+    import_error = e
+
+title = "Lindorm"
+name = "lindorm"
+type = HANDLER_TYPE.DATA
+
```

Comment:
Can we add the icon here? Please check other handlers

Reply:
icon for lindormdb /

Reply:
added into latest. commit